README.md

```diff
@@ -1,107 +1,118 @@
-# LAZY-DOTNET-ACTION
+# Lazy Action - Dotnet
 
-Setting up continuous integration 
+Setting up continuous integration
 
-- [LAZY-DOTNET-ACTION](#lazy-dotnet-action)
+- [Lazy Action - Dotnet](#lazy-action---dotnet)
   - [Prerequisites](#prerequisites)
     - [1. Setup github action workflow](#1-setup-github-action-workflow)
+    - [2. Add lazy action setup](#2-add-lazy-action-setup)
+    - [3. Inject lazy environment variables](#3-inject-lazy-environment-variables)
+
   - [Using Lazy Dotnet Action](#using-lazy-dotnet-action)
     - [Adding lazy dotnet action to workflow](#adding-lazy-dotnet-action-to-workflow)
-  - [Parameters](#parameters)
-    - [Input Parameters](#input-paramters)
-    - [Output Parameters](#output-paramters)
-
-
+  - [parameters](#parameters)
+    - [Input Parameters](#input-parameters)
 
 ## Prerequisites
 
 ### 1. Setup github action workflow
 
-
 1. On GitHub, navigate to the main page of the repository.
 2. Under your repository name, click Actions.
 3. Find the template that matches the language and tooling you want to use, then click Set up this workflow.Either start with blank workflow or choose any integration workflows.
 
+### 2. Add lazy action setup
+
+1. Add a code checkout step this will be needed to add code to the github workspace.
+
+```yaml
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+```
+
+2. This is to setup some of the global environment varibales we use as part of the lazy dotnet action.
+
+```yaml
+    - name: Setup
+      uses: variant-inc/lazy-action-setup@feature/init-setup
+```
+
+### 3. Add lazy action setup
```

Comment:
Point 2 and 3 says the same header.
I believe this isn't required

I think point 3 should be adding the lazy-action-dotnet.

Reply:
Corrected.